docs/teleport/aks-getting-started.md

```diff
@@ -0,0 +1,314 @@
+
+az acr import \
+  --source mcr.microsoft.com/oss/bitnami/redis:6.0.8 \
+  --name $ACR \
+  --image redis:6.0.8
+```
+
+## Confirm import and teleport expansion
+
+Confirm the `azure-vote-front` repository is set for teleport expansion:
+
+```azurecli
+az acr repository show \
+  --repository azure-vote-front \
+  -o jsonc
+```
+
+Look for `"teleportEnabled": true,` in the output
+
+```json
+"changeableAttributes": {
+  "deleteEnabled": true,
+  "listEnabled": true,
+  "readEnabled": true,
+  "teleportEnabled": true,
+  "writeEnabled": true
+}
+```
+
+Although the repository is configured for teleport expansion, each image upload will take time to be expanded on push. The length of time is based on the quantity and size of layers, however the expansion should be completed within seconds.
+
+> **Note:** ACR webhooks indicate when an artifact is pushed and available. A new webhook notification will be added at a future date to indicate the image has been expanded, and ready for teleportation.
+
+At this point in the Teleport preview, check expansion using the `check-expansion.sh` script. As the script uses a `/mount` api, basic auth is required. An [ACR Token](https://aka.ms/acr/tokens) is created and saved as environment variable.
+
+```azurecli-interactive
+export ACR_USER=teleport-token
+export ACR_PWD=$(az acr token create \
+  --name teleport-token \
+  --registry $ACR \
+  --repository azure-vote-front \
+  content/read \
+  --query credentials.passwords[0].value -o tsv)
+
+../check-expansion.sh teleport azure-vote-front v1
+```
+
+## Add nodes for teleporters and shuttles
+
+Two nodepools will be added to enable teleportation, and a comparison for standard transport. The system nodepool is avoided to enable clearing the image cache by scaling the nodepool to zero then back to one.
+
+```azurecli-interactive
+az aks nodepool add \
+  --resource-group $AKS_RG \
+  --cluster-name $AKS \
+  --name teleporter \
+  --node-count 1 \
+  --aks-custom-headers EnableACRTeleport=true \
+  --labels acr-teleport=enabled
+
+az aks nodepool add \
+  --resource-group $AKS_RG \
+  --cluster-name $AKS \
+  --name shuttle \
+  --node-count 1 \
+  --labels acr-teleport=disabled
+```
+
+## Deploy to AKS
+
+Update the `azure-vote-teleport.yaml` and `azure-vote-shuttle.yaml` files to reference your registry name:
+
+```yml
+spec:
+  ...
+  containers:
+  - name: azure-vote-back
+    image: <registryName>.azurecr.io/redis:6.0.8
+  ...
+  containers:
+  - name: azure-vote-front
+    image: <registryName>.azurecr.io/azure-vote-front:v1
+```
+
+### Deploy with standard pull performance
+
+Deploy the _shuttle_ podspec:
+
+```azurecli-interactive
+kubectl apply -f azure-vote-shuttle.yaml
+```
+
+Get the list of pods to find the azure-vote-front pod. The shorthand version can be used if only one pod is named `azure-vote-front`. You may need to run the command a few times until the image has been pulled and expanded on the node. `Successfully pulled image...`.
+
```

Comment:
Is `Successfully pulled image...` needed above?

Reply:
removed